tools/packaging/static-build/shim-v2/build.sh

```diff
@@ -60,10 +60,13 @@ sudo docker run --rm -i -v "${repo_root_dir}:${repo_root_dir}" \
 	bash -c "git config --global --add safe.directory ${repo_root_dir} && make PREFIX="${PREFIX}" DESTDIR="${DESTDIR}"  ${EXTRA_OPTS} install"
 
 sudo sed -i -e '/^initrd =/d' "${DESTDIR}/${PREFIX}/share/defaults/kata-containers/configuration-qemu.toml"
-sudo sed -i -e '/^initrd =/d' "${DESTDIR}/${PREFIX}/share/defaults/kata-containers/configuration-fc.toml"
+fc_config_file="${DESTDIR}/${PREFIX}/share/defaults/kata-containers/configuration-fc.toml"
+if [ -f ${fc_config_file} ]; then
+	sudo sed -i -e '/^initrd =/d' ${fc_config_file}
+fi
```

Comment:
Do you care about firecracker for s390x?

If not, you're doing a change here in a file that will be removed down below.

Reply:
Choi can correct me if I'm wrong, but IIRC I think the firecracker config wasn't there for the s390x build:
```
sed: can't read /home/jenkins/workspace/kata-containers-CCv0-tests-ubuntu-20.04-s390x-PR/go/src/github.com/kata-containers/kata-containers/tools/packaging/kata-deploy/local-build/build/cc-shim-v2/destdir//opt/confidential-containers/share/defaults/kata-containers/configuration-fc.toml: No such file or directory
```

So this code is to skip it and stop that failure, but not impact the platforms that firecracker is there?

Reply:
Yes, @stevenhorsman is correct. This is just a pre-check to carry out the relacement. The file does not exist for s390x. But @hbrueckner could answer if we care about firecracker.

Reply:
Ah, I see.  I missed this file was already present.

@BbolroC, for consistency sake, could you do the same for the QEMU file?

Reply:
yes, I will. thanks for the comment. 😉

Reply:
> yes, I will. thanks for the comment. wink

Maybe it is also worth to have both file then process in the `for` loop like `for vmm in qemu fc`, `test` and perform the `sed`.

Otherwise than this:

LGTM